File: Baekjoon/10101 - 삼각형 외우기/10101.cpp
#include <iostream>
using namespace std;

#define FASTIO cin.tie(NULL); cout.tie(NULL); ios::sync_with_stdio(false);

int main() {
    int a, b, c;

    cin >> a >> b >> c;

    if(a == 60 && b == 60 && c == 60) cout << "Equilateral";
    else if(a+b+c == 180 && (a == b || b == c || c == a)) cout << "Isosceles";
    else if(a+b+c == 180 && !(a == b || b == c || c == a)) cout << "Scalene";
    else cout << "Error";
}

File: solution/10xxx/10101 - 삼각형 외우기/10101.py
import sys
input = sys.stdin.readline

triangle = list(int(input()) for _ in range(3))

if triangle == [60, 60, 60]:
    print("Equilateral")
elif triangle[0] + triangle[1] + triangle[2] == 180:
    if triangle[0] == triangle[1] or triangle[1] == triangle[2] or triangle[2] == triangle[0]:
        print("Isosceles")
    else:
        print("Scalene")
else:
    print("Error")